python_modules/dagster/dagster/core/definitions/repository_definition.py

```diff
@@ -603,6 +605,12 @@ def from_dict(repository_definitions: Dict[str, Dict[str, Any]]) -> "CachingRepo
 
             if isinstance(job, GraphDefinition):
                 repository_definitions["jobs"][key] = job.coerce_to_job()
+            elif isinstance(job, UnresolvedAssetJobDefinition):
+                repository_definitions["jobs"][key] = job.resolve(
+                    # TODO: once AssetDefs are on the repository, do this
```

Comment:
nit bc I know this is going to be resolved shortly: but github issue is more trackable than TODO